### DISCUSSION TITLE
Design systems and identifiers

### DISCUSSION THREAD

**Discussion by Naoto-Ida** [vanilla-extract-css/vanilla-extract#821] [Q&A]
Hi y'all, I just wanted to firstly say that I love Vanilla Extract! My team is considering adoption of it for our new design system.

One question that comes to mind is how to deal with [identifiers](https://vanilla-extract.style/documentation/integrations/vite/#identifiers) (will use the Vite configuration page as an example).
Our design system will be used in various forms; public-facing websites, internal web apps, etc.
We want to retain the "debug" identifier for when publishing the design system to be consumed so that if there are any issues, our teams can investigate easier with the debug format. But we **also** want to give the individual apps the option to use the "short" identifier for when they deploy.

Currently our mock design system has the default "identifiers" value of "debug".
When adding this DS as a dependency to a demo app, I've attempted to add the relevant VE plugin to the build tool and set the "identifiers" value to "short". This does not seem to take effect.

Am I missing something from the documentation? Is this something that can be tackled out of the scope of VE?

**Comment by mattcompiles** [comment]
Hi @Naoto-Ida 👋

Are you compiling the DS to CSS files before consuming it? Once the vanilla-extract code is compiled to CSS there's no changing it.

There are other options though.

**Reply by Naoto-Ida** [reply]
Is this other way referring to what Braid Design System does, which exports TS files directly?
